src/Service/UserCreator.php

```diff
@@ -44,28 +45,19 @@ public function create( $username, $password, $email = null ) {
 		}
 
 		$params = [
-			'name' => $username,
+			'createreturnurl' => $this->api->getApiUrl(),
```

Comment:
So, what version of MediaWiki does this work back until? :O

Reply:
It might be nice to automatically detect if this isn't a valid api call and fallback to the old method, we do this somewhere else in these libraries too, not sure to what extent we should really.

https://github.com/addwiki/mediawiki-api-base/blob/6fb0381a6095f6edabe3692342890a3b710be7b7/src/MediawikiSession.php#L73

But if it is just login / create account stuff I think we should make the effort to support the older versions!

Reply:
I think we should support all current versions as defined in https://www.mediawiki.org/wiki/Version_lifecycle — so 1.27 and higher. Or is that too strickt?

I've added the extra MW versions to test against.

Reply:
This works with pre-1.27 now.